docs/third-party-slick-3.md

```diff
@@ -33,6 +33,8 @@ and on [Stackoverflow](http://stackoverflow.com/questions/tagged/slick).
 
 ## Example projects
 
+- [Template Site](https://github.com/tetrao-eu/template-site) Skeleton of a web application based on Scala, SBT, Play Framework, Play2-auth, PostgreSQL, Slick, Slick-pg and Foundation [https://template-site.tetrao.eu/]
```

Comment:
The link at the end of the line is rendering as...

![selection_001](https://cloud.githubusercontent.com/assets/102661/16193474/45496164-36e7-11e6-9a16-84fc69b7a7f9.png)